### PR TITLE
updated picture size using html and updated boilerplate

### PR DESCRIPTION
**Pull request by victorialherring**
sorry my branch name doesn't make sense it was an accident lol

### CODE REVIEW

File: personal.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Deadmau5 4Ev3r</title>
+    <link rel="stylesheet" type="text/css" href="css/main.css">
```
Comment (dLars99) [inline personal.html]: Correct "main.css" to "styles.css"

**Review by dLars99 — CHANGES_REQUESTED** [review]
Please correct "main.css" to "styles.css"

**Review by dLars99 — APPROVED** [review]

**Comment by chrismccolgan** [comment]
tes

**Review by chrismccolgan — APPROVED** [review]
yes

**Comment by victorialherring** [comment]
MERGING

**Pull request merged**